strings.md

```diff
@@ -1009,6 +1010,31 @@ With `~2,2f`:
 
 And we're happy with this result.
 
+### Iteration
+Create a string from a list with iteration construct `~{str~}`:
+~~~lisp
```

Comment:
and also add a newline before the `~~~lisp` code snippet, it is necessary for the epub generation.

Necessary for all the snippets below.